### DISCUSSION TITLE
How to disable copyright?

### DISCUSSION THREAD

**Discussion by MonoLogueChi** [vuepress-theme-hope/vuepress-theme-hope#2745] [Q&A] [answered]
I am using 172

When I use `theme.copyright = false` the warning appears `copyright is deprecated in V2, please use plugins.copyright instead.`, but I found that `plugins.copyright` seems to be a plugin about copying and adding copyright, not the configuration of the footer

**Comment by Mister-Hope** [comment, marked as answer]
Fixed in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/78b3cf21bb695a295a22715d5e41eb47037b7eee
